CHANGELOG.md

```diff
@@ -2,13 +2,11 @@
 
 ## Versions from 0.4x
 
-### Ongoing
-
-- Fix building (remaining plugwise instead of plugwise_usb)
-
 ### 0.40.5
 
-- Update backend-library to v0.31.2b6: fix stuck energy_consumption_today, provided by @dirixmjm
+- Fix building (remaining plugwise instead of plugwise_usb).
+- Update backend-library to v0.31.2: fix sequence_id counter skew, provided by @dirixmjm.
+- Update backend-library to v0.31.2b6: fix stuck energy_consumption_today, provided by @dirixmjm.
```

Comment:
Please, 2x, remove the "Update backend-library to v0.31.x". The part after the ":" provides sufficient info.